README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">BPMN Visualization</h1>
 <div align="center">
-    <p align="center"> <img title="BPMN Visualization" src="docs/users/images/custom-behavior-path-highlighting.gif" alt="BPMN Visualization example"></p>
+    <p align="center"> <img title="BPMN Visualization" src="https://github.com/process-analytics/bpmn-visualization-examples/master/examples_home_for_hacktoberfest.png" alt="BPMN Visualization example"></p>
```

Comment:
The url doesn't display an image, I get a 404 error. It should be something like: https://raw.githubusercontent.com/process-analytics/bpmn-visualization-examples/master/examples_home_for_hacktoberfest.png
I suggest we don't ref the master branch but a commit to ensure we have a fully working url over the time. We target a specific version of the image, so a specific commit is fine for that.

So my proposal is to use: https://github.com/process-analytics/bpmn-visualization-examples/raw/a64e681763923c4161b774326936afdebc367c18/examples_home_for_hacktoberfest.png

Reply:
It's weird. I tested from this PR, and the image was displayed.
But, I changed it.